internal/consoleplugin/reconciler.go

```diff
@@ -229,6 +230,8 @@ func (r *Reconciler) mutateDeployment() error {
 								Protocol:      "TCP",
 							},
 						},
+						Command: []string{"plugin-backend"},
+						Args:    []string{"-port", "9002", "-features", strings.Join(r.Config.Features, ",")},
```

Comment:
Nginx was replaced with the go backend. So here we need to add the certificate and key args that the new backend supports: https://github.com/openshift/logging-view-plugin/pull/55/commits/a39036acf1dad1cd0ec28bdd6ef055697db5b1db#diff-4d25889b839b563a7b3536730d4aa8bf08b34a5fa25702dc3bfcd9dbfdca444cR11, also the port should be `9443`